src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,44 @@
+import { Link, useParams } from "react-router-dom";
+import { Person } from "../../types"
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr data-cy="person"
+      className={slug === person.slug  ? "has-background-warning" : ''}>
```

Comment:
Use the classnames library for add classes with condition, fix it everywhere
```suggestion
    <tr data-cy="person" className={slug === person.slug  ? "has-background-warning" : ''}>
```